### PR TITLE
MTP-1818: Ability to accept/reject Account Requests

### PR DESCRIPTION
**Pull request by xoen**
- FIU see a banner with the number of 'security' account requests
  (when there are any)
- FIU can manage users and accept/reject these account requests

Ticket: https://dsdmoj.atlassian.net/browse/MTP-1818

### CODE REVIEW

File: mtp_noms_ops/templates/security/includes/account-requests-banner.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <a href="{% url 'list-users' %}">
+            {% blocktrans trimmed count count=user_request_count %}
+                You have {{ count }} new account request
+            {% plural %}
+                You have {{ count }} new account requests
+            {% endblocktrans %}
+        </a>
+    {% endcaptureoutput %}
+
+    {% include 'mtp_common/components/notification-banner.html' with banner_level='important' banner_title=_('Important') banner_heading=account_requests_heading only %}
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply mtp_noms_ops/templates/security/includes/account-requests-banner.html:16]: i.e. you don't need to pass in `banner_level` as you're not changing the appearance of the box

[the remark this answers is not in the document]
Comment (xoen) [reply mtp_noms_ops/templates/security/includes/account-requests-banner.html:16]: It seemed to not cause problems anyway but I'm removing it, less is more 👍 

Speaking of, [`banner_title` also defaults to `_('Important')`](https://github.com/ministryofjustice/money-to-prisoners-common/blob/master/mtp_common/templates/mtp_common/components/notification-banner.html#L8) so I'm removing it as well.

[the remark this answers is not in the document]
Comment (xoen) [reply mtp_noms_ops/templates/security/includes/account-requests-banner.html]: FYI: https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/534/commits/e3f4467d719b73236d8ca955c3aa919a9646fee3

**Review by ushkarev — APPROVED** [review]

**Comment by xoen** [comment]
FYI @ushkarev I've rebased this.

**Review by ushkarev — APPROVED** [review]

**Comment by xoen** [comment]
Thanks @ushkarev ☺️

**Pull request merged**